app/src/main/java/org/grapheneos/apps/client/utils/network/MetaDataHelper.kt

```diff
@@ -224,6 +227,21 @@ class MetaDataHelper constructor(context: Context) {
         return result
     }
 
+    private fun isSystemApp(packageName: String, ogPackageName: String? = null): Boolean {
+
+        val sigFlags = PackageManager.GET_SIGNING_CERTIFICATES
```

Comment:
Why obtain the signing certificates? They aren't used in this method.

Reply:
fixed.